modules/Oculus_module/include/OculusModule.hpp

```diff
@@ -244,6 +247,52 @@ class OculusModule : public yarp::os::RFModule
      * @return true in case of success and false otherwise.
      */
     bool close() final;
+
+    /**
+     * This method prepares the Oculus Module for Teleoperation.
+     * @return true/false in case of success/failure;
+     */
+    virtual bool prepareTeleoperation() override;
+
+    /**
+     * This method runs the entire Oculus Module for Teleoperation.
+     * @return true/false in case of success/failure;
+     */
+    virtual bool runTeleoperation() override;
+
+    /**
+     * This method is called in order to prepare the Oculus Module FSM for Teleoperation.
+     * @return true/false in case of success/failure;
+     */
+    bool preparingModule();
+
+    /**
+     * This method is called in order to run the Oculus Module FSM for Teleoperation.
+     * @return true/false in case of success/failure;
+     */
+    bool runningModule();
+
+    /**
+     * Respond to a message from the RPC port.
+     * @param command is the received message.
+     * The following message has to be a bottle with the following structure:
+     * 1. ("help");
+     * 2. ("prepare").
+     * 3. ("run")
+     * @param reply is the response of the server.
+     * 1. OK in case of success;
+     * 2. Not OK in case of failure.
+     * 3. list of available commands in case of "help" command
+     * @return true in case of success and false otherwise.
+     */
+    bool respond(const yarp::os::Bottle& command, yarp::os::Bottle& reply) final;
```

Comment:
I think this is not necessary anymore

Reply:
Yes, I should remove it, it is a left over, thanks.